hadoop-cloud/src/test/scala/org/apache/spark/internal/io/cloud/StubPackageEntry.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.internal.io.cloud
+
+/**
+ * This class is here to make sure that the source tree is found and built.
```

Comment:
Hmm, I'm not following why this class is here.

Reply:
it's a funny with maven. If you don't have a source file in the non-3.2-profile source tree, that 3.2 source tree doesn't get built. Took at while to work out and doesn't actually deliver anything of value in production code

Reply:
I tried what you say locally (I deleted the `hadoop-cloud/src/test` directory), and all the hadoop-3.2 stuff was built, and tests ran.

Reply:
ok, seems to work for me too, once I can get maven past some hangings elsewhere in the build. Maybe a recent change in the compilers.

Reply:
...cutting the `test/src/scala` tree, leaving only `test/resources/log4j.properties`